Automattic/studio
# Add AI Assistant tab

Resolves Automattic/dotcom-forge#7322

## Proposed Changes

- Adds a new AI Assistant tab behind a feature flag
- Adds placeholder tests for future AI Assistant tab, and tests for feature flag


## Testing Instructions

The AI Assistant tab can only be tested behind a temporary feature flag. 

To test:
- Check out the `feat/ai-assistant-tab` branch locally.
- Run `STUDIO_AI=true npm start` to enable the feature flag and render the Assistant tab. (Or any other alternative for setting process environment variables, e.g. `$env:STUDIO_AI="true"; npm start` when using PowerShell.)
- Run `npm start` to hide the Assistant tab. The app should appear as it does in `trunk`.

> [!NOTE]  
> As indicated on Automattic/dotcom-forge#7322, the content of the Assistant tab itself is **placeholder content**. The actual design of the tab is still in progress, and will be carried out in https://github.com/Automattic/dotcom-forge/issues/7323.

**macOS**
<img width="1012" alt="macOS" src="https://github.com/Automattic/studio/assets/643285/031f8070-71a1-48bd-a547-eb6a947491ed">

**Windows**
<img width="1023" alt="windows" src="https://github.com/Automattic/studio/assets/643285/d7757fe7-0968-4e80-95fe-0a04f82c042b">

**RTL Layout** (Note: the locale displayed below is an example, and is currently untranslated)
<img width="1012" alt="rtl" src="https://github.com/Automattic/studio/assets/643285/630ec039-7e50-4bb8-b795-6746ae26661c">


## Pre-merge Checklist
